samples/samples-aspnet/embedded-sign-in-widget/embedded-sign-in-widget-e2etests/Steps/BasicLoginWithPasswordFactorSteps.cs

```diff
@@ -0,0 +1,36 @@
+﻿using embedded_sign_in_widget_e2etests.Helpers;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using TechTalk.SpecFlow;
+
+namespace embedded_sign_in_widget_e2etests.Steps
+{
+    [Binding]
+    public class BasicLoginWithPasswordFactorSteps : BaseTestSteps
+    {
+
+        public BasicLoginWithPasswordFactorSteps(ITestContext context)
+            : base(context)
+        {
+            context.TestUserProfile = new TestUserProfile
+            {
+                Email = "testuser@threeheadz.com",
+                Password = "Abcd1234"
```

Comment:
IIRC we don't want this particular password string to be exposed. It may be worth moving it to env vars or changing to something else like `Abcd12345`